### PR TITLE
210 ml

### PR DESCRIPTION
**Pull request by maurolepore**
- Use relationship many to many
- Restore test about thresholds
- Flag fixme

### CODE REVIEW

**Review by kalashsinghal — COMMENTED** [review]
@maurolepore Thanks for correcting the test cases in this PR! Your PR looks great to me. I will merge this PR after resolving the conflicts.

File: R/pctr.R [hunk 1]
```diff
@@ -52,7 +52,12 @@ pctr_at_product_level <- function(companies,
...
     pctr_add_scores(low_threshold, high_threshold)
 
-  left_join(companies, scored, by = "activity_uuid_product_uuid")
+  left_join(
+    companies,
+    scored,
+    by = "activity_uuid_product_uuid",
+    relationship = "many-to-many"
+  )
 }
```
Comment (kalashsinghal) [inline R/pctr.R:61]: @maurolepore why the parameter `relationship = "many-to-many"` is introduced? I guess the `left_join` function will automatically do a `many-to-many` join. right?

**Review by kalashsinghal — COMMENTED** [review]
@maurolepore LGTM. I will fix the bug of `high_threshold` in the separate issue you assigned me.

File: R/pctr.R (hunk 1 above)
Comment (maurolepore) [reply R/pctr.R:61]: It avoids this warning:

```r
Warning (test-pstr_at_product_level.R:11:3): added column returns acceptable values
Detected an unexpected many-to-many relationship between `x` and `y`.
i Row 1 of `x` matches multiple rows in `y`.
i Row 49 of `y` matches multiple rows in `x`.
i If a many-to-many relationship is expected, set `relationship = "many-to-many"` to silence this warning.
```

**Comment by maurolepore** [comment]
RE Failing checks: Don't worry, I'll fix it.

Likely it's because we merged in the wrong order. 

The correct order is this: 
```
main < 2. your-pr < 1. my-pr
```
So `my-pr` merges into `your-pr`, then `your-pr` (now with my commits) merges into `main`

**Comment by maurolepore** [comment]
I'll follow up on #220

**Pull request closed without merging**